Image Encryption AES/AESFunctions.cpp

```diff
@@ -433,7 +432,7 @@ void aes::sBoxSubstitution(unsigned char* const& buffer, const std::size_t buffe
 
 void aes::printBufferRowMajorOrder(const unsigned char* const& buffer, const std::size_t size, const std::size_t colCount)
 {
-    assert(size % colCount == 0);
+    assert(size % colCount == 0);  // TODO (FIX): This assertion fails when a 17-character key is given on the command line
```

Comment:
@Andrew-Miner This assert failed when I used a 17-byte field (e.g. "asdfasdfasdfasdfx")

Reply:
Fixed